parity-util-mem/Cargo.toml

```diff
@@ -11,6 +11,8 @@ edition = "2018"
 cfg-if = "0.1.10"
 dlmalloc = { version = "0.1.3", features = ["global"], optional = true }
 wee_alloc = { version = "0.4.5", optional = true }
+lru = { version = "0.4" }
```

Comment:
Do you think it makes sense to make them `optional = true`, but include into the `default` features, so that they can be opt-out?

Reply:
Yep